packages/@romejs/core/test-worker/TestAPI.ts

```diff
@@ -667,6 +667,10 @@ export default class TestAPI implements TestHelper {
 		});
 	}
 
+	getNextSnapshotId(): number {
+		return this.snapshotCounter++;
+	}
```

Comment:
I don't like this API. I think it's better to change it to something like this:
```suggestion
	getLastSnapshotId(): number {
		return this.snapshotCounter;
	}
```

And then to maintain your own counter or have something else incrementing the counter but not a getter.

Reply:
Sure, I can change it.

`t.snapshot` currently uses `String(this.snapshotCounter++)` as the `entryName`.

I want some way of creating a custom entryName that still participates in test-level numbering. I thought about adding some sort of label field to `TestSnapshotOptions` that `t.snapshot` could interpolate into the `entryName`, but I thought that it would be more flexible if you could just use `t.namedSnapshot`

Are you ok with an `incrementSnapshotCounter()` method in addition to the `getLastSnapshotId()` method?

I wasn't sure about creating a mutating getter but then I saw `getNextWorkerId` in `WorkerManager` and I thought "When in Rome..."

Reply:
Having thought about it, I don't think that the extra flexibility is worth bloating the TestAPI with two new methods.

The main reason that I wanted a way for helpers like `testLint` to have access to `snapshotCounter` is that entries are sorted in the snapshot by entryName within each `test()`. Using test-level numbering is a simple way of making sure that the entries created by helpers show up in the order you called those functions.

But I suppose that if you really care about the order of snapshot entries, you can write more than one `test()` in your file.